composer.json

```diff
@@ -2,16 +2,11 @@
     "name":"magento/magento-composer-installer",
     "description":"Composer installer for Magento modules",
     "keywords":[ "composer-installer", "magento"],
-    "minimum-stability":"stable",
+    "minimum-stability":"dev",
```

Comment:
should it be reverted ?

Reply:
hi @xmav , thanks for the note, reverted